static/docs/command-reference/version.md

```diff
@@ -64,7 +64,7 @@ The detail of `Binary` depends on the way DVC was downloading and
     distributions, like Ubuntu
   - Red Hat package (`.rpm`) - file used to install packages in some Linux based
     distributions, such as Fedora, CentOS, etc.
-  - PKG file (`.pkg`) - file used to install apps on macOS
```

Comment:
I think `macOS` is the right way actually

Reply:
I think either way is correct. macOS is more of a logo/brand design, MacOS is the correct way to spell a proper noun. My change was more for consistency's sake.

![image](https://user-images.githubusercontent.com/1477535/67153786-46a91800-f2b5-11e9-85dd-2fe7dca4e892.png)

Reply:
but lmk if you prefer macOS I'll replace all.

Reply:
no, I'm fine with either :)